tests/sles4sap/sap_deployment_automation_framework/execute_playbooks.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary:  Executes list of playbooks compiled according to defined scenario.
+#           https://learn.microsoft.com/en-us/azure/sap/automation/tutorial#sap-application-installation
+# Playbooks can be found in SDAF repo: https://github.com/Azure/sap-automation/tree/main/deploy/ansible
+
+use parent 'sles4sap::sap_deployment_automation_framework::basetest';
+use strict;
+use warnings;
+use sles4sap::sap_deployment_automation_framework::deployment
+  qw(serial_console_diag_banner
+  load_os_env_variables
+  az_login
+  sdaf_execute_playbook
+  ansible_hanasr_show_status
+  ansible_ensa2_show_status
+  playbook_settings
+  );
```

Comment:
At this time wouldn't it be better to simply do:
```
use sles4sap::sap_deployment_automation_framework::deployment;
```
(Assuming all the functions listed above are exported by the package)

Reply:
Yeah. I was even considering it while adding another import.